notebooks/community/managed_notebooks/sentiment_analysis/Sentiment_Analysis.ipynb

```diff
@@ -0,0 +1,1168 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    labels = [0, 1, 2, 3]
</pre><p>I think the notebook would be more readable if you defined constants at the top, and then referred to them instead of 0, 1, 2, 3. (Note that this comment applies to multiple cells throughout the notebook.)</p><p> </p><p>For example,</p><pre class="ql-syntax" spellcheck="false">NEGATIVE=0
NEUTRAL=1
POSITIVE=2
# ???=3 (not sure what 3 is)
</pre><p> </p><p>Here, you'd use</p><pre class="ql-syntax" spellcheck="false">labels=[NEGATIVE, NEUTRAL, POSITIVE, ...]
</pre><p> </p><p>Later, you could do things like:</p><p> </p><pre class="ql-syntax" spellcheck="false">subset_data = df[df["label"].isin([NEGATIVE, ???])].reset_index(drop=True)
</pre><p>instead of</p><pre class="ql-syntax" spellcheck="false">subset_data = df[df["label"].isin([0, 3])].reset_index(drop=True)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
resolved